### PR TITLE
Update the hyperlink address of High-Availability Clusters

### PR DESCRIPTION
**Pull request by vaibhav2107**
Ref #27094

### CODE REVIEW

File: content/zh/docs/concepts/overview/components.md
Comment (jihoon-seo) [inline content/zh/docs/concepts/overview/components.md:65]:
```suggestion
请参阅[构建高可用性集群](/zh/docs/setup/production-environment/tools/kubeadm/high-availability/)
```

File: content/ja/docs/concepts/overview/components.md
Comment (jihoon-seo) [inline content/ja/docs/concepts/overview/components.md:31]:
```suggestion
マルチマスター VMセットアップの例については、[高可用性クラスターの構築](/ja/docs/setup/production-environment/tools/kubeadm/high-availability/) を参照してください。
```

File: content/fr/docs/concepts/overview/components.md
Comment (jihoon-seo) [inline content/fr/docs/concepts/overview/components.md:25]:
```suggestion
Voir [Construire des Clusters en Haute Disponibilité](/fr/docs/setup/production-environment/tools/kubeadm/high-availability/) pour une configuration d'exemple en multi-master-VM.
```

File: content/id/docs/concepts/overview/components.md
Comment (jihoon-seo) [inline content/id/docs/concepts/overview/components.md:33]:
```suggestion
dapat dilihat di modul [Membangun Klaster HA] (/id/docs/setup/production-environment/tools/kubeadm/high-availability/).
```

File: content/id/docs/reference/glossary/kube-apiserver.md
Comment (jihoon-seo) [inline content/id/docs/reference/glossary/kube-apiserver.md:18]:
```suggestion
Komponen ini didesain agar dapat diskalakan secara horizontal. Lihat [Membangun Klaster HA](/id/docs/setup/production-environment/tools/kubeadm/high-availability/).
```

File: content/id/docs/setup/best-practices/multiple-zones.md
Comment (jihoon-seo) [inline content/id/docs/setup/best-practices/multiple-zones.md:77]:
```suggestion
tinggi harus mengikuti instruksi [ketersediaan tinggi](/id/docs/setup/production-environment/tools/kubeadm/high-availability/).
```

File: content/fr/docs/reference/glossary/kube-apiserver.md
Comment (jihoon-seo) [inline content/fr/docs/reference/glossary/kube-apiserver.md:18]:
```suggestion
Il est conçu pour une mise à l'échelle horizontale, ce qui veut dire qu'il met à l'échelle en déployant des instances supplémentaires. Voir [Construire des Clusters en Haute Disponibilité](/fr/docs/setup/production-environment/tools/kubeadm/high-availability/).
```

**Review by jihoon-seo — CHANGES_REQUESTED** [review]
Thank for your contribution.
I suggested some changes, since there are some translated versions of the target page. :)

**Comment by jihoon-seo** [comment]
- This PR resolves #27094

**Review by sftim — COMMENTED** [review]
@vaibhav2107 we try to limit fixes to a single language per PR - see https://kubernetes.io/docs/contribute/new-content/overview/#languages-per-pr

This change would be difficult to approve as it requires co-ordination with more than 10 localization teams. Could you update just the English version for now?

Once the English change merges, we'd be very happy to review follow-up PRs to update the relevant localized pages.

**Comment by sftim** [comment]
This might be OK to merge but getting all the approvals will be tricky!

**Comment by seokho-son** [comment]
As a Korean contents owner, LGTM :) 
But I would like to postpone giving approval to this PR until @vaibhav2107  's feedback on the comment from @sftim .

**Comment by tengqm** [comment]
There is no change to zh localization although `language/zh` is added.
Good luck with this PR spanning so many teams.

**Comment by vaibhav2107** [comment]
> @vaibhav2107 we try to limit fixes to a single language per PR - see https://kubernetes.io/docs/contribute/new-content/overview/#languages-per-pr
> 
> This change would be difficult to approve as it requires co-ordination with more than 10 localization teams. Could you update just the English version for now?
> 
> Once the English change merges, we'd be very happy to review follow-up PRs to update the relevant localized pages.

Thank You @sftim for informing about this. Now on, I'll be careful not to fix more than one language in a PR.

**Comment by atoato88** [comment]
@vaibhav2107 
As @sftim says, could you update this PR to include changes for English contents only?
I think target files of this PR are following:
- content/en/docs/concepts/overview/components.md 
- content/en/docs/tasks/debug-application-cluster/debug-cluster.md

That's say remove other files (which include contents for other language localization) from this PR and those files will be updated in other PRs.

And also rebase is needed for current PR codeset.

**Comment by jihoon-seo** [comment]
The link in the English version was updated in the PR #26725.
This PR seems good to close.

**Comment by atoato88** [comment]
@vaibhav2107 
This PR has `content/en/docs/tasks/debug-application-cluster/debug-cluster.md` also.
So it's good to update PR with including `content/en/docs/tasks/debug-application-cluster/debug-cluster.md` only, and update files on multi language next on other PR.
WDYT?

**Comment by atoato88** [comment]
I think update flow is following:
- At first, this PR needs rebase with latest `kubernetes:master`.
- Next, update PR by excluding files except for `en` contents.
- Third, exclude `content/en/docs/concepts/overview/components.md` because #26725 fixed this file already.

**Comment by atoato88** [comment]
Thank you for updating. :smile: 
PR includes [`content/en/docs/concepts/overview/components.md` yet](https://github.com/kubernetes/website/pull/27139/files#diff-5aa7dbbe8bb73f90af59eb44d3118360ebe22fede8a50d99fedb3d9c197ef0f2R36-R41), so you need to exclude this file from PR.

**Comment by atoato88** [comment]
And I remove tags which doesn't related to current codeset.

/remove-language fr 
/remove-language id 
/remove-language it 
/remove-language ja 
/remove-language ko 
/remove-language pl 
/remove-language pt
/remove-language ru
/remove-language zh
/remove-language de
/remove-language es

**Pull request merged**
